packages/evm/src/evm.ts

```diff
@@ -251,7 +251,7 @@ export class EVM implements EVMInterface {
     // Load code
     await this._loadCode(message)
     let exit = false
-    if (!message.code || message.code.length === 0) {
+    if (!message.code || (typeof message.code !== 'function' && message.code.length === 0)) {
```

Comment:
Happy to add a test if we want one

Reply:
Really interesting find 🙏 and usage! Yes, it would be great if you can take the relevant parts of the test case over.

Reply:
Apologies GitHub notifications are shambles I didn’t see this comment